Source/EventAnalysis/KTOverlappingTrackClustering.cc

```diff
@@ -143,6 +207,40 @@ namespace Katydid
 
         return true;
     }
+    bool KTOverlappingTrackClustering::FindMatchingSeqLineCands()
```

Comment:
Template methods are a thing!
Maybe you can use that to simplify this file a bit.